src/battle-tooltips.ts

```diff
@@ -814,7 +814,8 @@ class BattleTooltips {
 				if (pokemon.status === 'tox') {
 					if (pokemon.ability === 'Poison Heal' || pokemon.ability === 'Magic Guard') {
```

Comment:
```suggestion
					const damageIncreaseStep = pokemon.dynamax ? 100 / 8 : 100 / 16;
					const toxicDamage = Math.floor(damageIncreaseStep * Math.min(clientpokemon.statusData.toxicTurns + 1, 15);
					if (pokemon.ability === 'Poison Heal' || pokemon.ability === 'Magic Guard') {
```

Reply:
This is what I mean by using another variable (you need to use this in the two remaining if branches for Poison Heal/Magic Guard and regular damage).
```suggestion
					let damageIncreaseStep = 100 / 16;
					if (pokemon.dynamax) damageIncreaseStep /= 2;
					const toxicDamage = Math.floor(damageIncreaseStep * Math.min(clientpokemon.statusData.toxicTurns + 1, 15));
					if (pokemon.ability === 'Poison Heal' || pokemon.ability === 'Magic Guard') {
```